src/js/vre/chapter31/components/EducationPeriodView.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default function EducationView({ formData }) {
+  let from = '';
+  let to = '';
+  if (formData.yearStarted && formData.yearLeft) {
+    from = formData.yearStarted;
+    to = formData.yearLeft;
+  }
+
+  return (
+    <div>
+      <strong>{formData.program}</strong><br/>
+      {from} &mdash; {to}
```

Comment:
Since the to and from fields are optional, displaying the dash should probably be conditional. Maybe generate the whole string with a single template assignment around line 7.

Reply:
Good idea, changed.